frontend/public/components/factory/modal.tsx

```diff
@@ -90,7 +90,9 @@ export const ModalTitle: React.SFC<ModalTitleProps> = ({
 export const ModalBody: React.SFC<ModalBodyProps> = ({ children }) => (
   <div className="modal-body">
     <div className="modal-body-content">
-      <div className="modal-body-inner-shadow-covers">{children}</div>
+      <div className="modal-body-inner-shadow-covers">
```

Comment:
You could do `<TextContent className="modal-body-inner-shadow-covers">` to remove a node (assuming that doesn't break the shadows).

Reply:
yes it works